src/data/battler-tags.ts

```diff
@@ -140,6 +140,11 @@ export interface TerrainBattlerTag {
   terrainTypes: TerrainType[];
 }
 
+export interface RestrictingBattlerTag {
+  interruptedText: (pokemon: Pokemon, moveId: MoveId) => string;
+  selectionDeniedText: (pokemon: Pokemon, moveId: MoveId) => string;
```

Comment:
```suggestion
  getInterruptedText: (pokemon: Pokemon, moveId: MoveId) => string;
  getSelectionDeniedText: (pokemon: Pokemon, moveId: MoveId) => string;
```

Makes it clearer that these are functions, not attributes